web-ui/src/components/reports-section/CheckinReport.jsx

```diff
@@ -1,36 +1,188 @@
-import React, { useContext } from 'react';
+import React, { useContext, useState } from 'react';
 import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
 
 import { getAvatarURL } from '../../api/api.js';
 import { AppContext } from '../../context/AppContext';
 import { selectFilteredCheckinsForTeamMemberAndPDL } from '../../context/selectors';
 
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import ExpandMore from '../expand-more/ExpandMore';
+
 import {
   Accordion,
   AccordionDetails,
   AccordionSummary,
   Avatar,
   Box,
   Card,
-  CardHeader,
   CardContent,
+  CardHeader,
   Chip,
+  Collapse,
   Container,
+  Divider,
+  Stepper,
+  Step,
+  StepLabel,
   Typography
 } from '@mui/material';
 
+/**
+ * @typedef {Object} Checkin
```

Comment:
Should the types `Checkin`, `CheckinStatus`, and `SchedulingStatus` be moved to `app.d.js`?